support-frontend/assets/components/subscriptionCheckouts/subscriptionsSurvey/SubscriptionsSurvey.jsx

```diff
@@ -0,0 +1,43 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+
+import type { SubscriptionProduct } from 'helpers/subscriptions';
+
+import AnchorButton from 'components/button/anchorButton';
+import Text from 'components/text/text';
+import Content from 'components/content/content';
+import './subscriptionsSurvey.scss';
+
+type PropTypes = {|
+  product: SubscriptionProduct,
+|};
+
+const surveyLinks: {[key: SubscriptionProduct]: string} = {
+  DigitalPack: 'https://www.surveymonkey.co.uk/r/QF9ZGQR',
+  GuardianWeekly: 'https://www.surveymonkey.co.uk/r/QFNYV5G',
+  Paper: 'https://www.surveymonkey.co.uk/r/Q37XNTV',
+};
+
+export const SubscriptionsSurvey = ({ product }: PropTypes) => {
+  const surveyLink = surveyLinks[product];
+  const title = 'Tell us about your subscription';
```

Comment:
is this tell us more about your purchase or experience? as they only just started the subscription.